lib/active_shipping/carriers/fedex.rb

```diff
@@ -143,14 +143,15 @@ def requirements
     end
 
     def find_rates(origin, destination, packages, options = {})
+      pickup_date = options.delete(:pickup_date)
```

Comment:
Small thing, but I think it's best to move the `pickup_date = ...` line down beneath `options = ...` and `packages = ...` just because so many active shipping methods start with those two lines I think it's probably good to stick to that pattern